docs/course_materials/compound_data_types.html

```diff
@@ -404,15 +737,82 @@ <h3><b>Sequence operations</b></h3>
                         <li>For loop through a list:</li>
                     </ul>
                     <pre><code data-trim data-noescape class="hljs lang-python">
-for x in l:
-    print(x)
-# orange
-# apple
-# plum
-# banana
+                        my_list = ['orange', 'apple', 'plum', 'banana']
+                        for item in my_list:
+                            print(item)
+                        # orange
+                        # apple
+                        # plum
+                        # banana
                     </code></pre>
                 </section>
             </section>
+            <section>
+                <section>
+                    <h1><b>
+                            <span>Conclusion</span>
```

Comment:
Really liked the table!
Do we want to move it before the Extra slide without the set, and create another table after the set?

Reply:
It's possible, that's why I also left the verbal conclusion at the end.
I just don't want to confuse the students.